localization/english/culture/wc_culture_titles_l_english.yml

```diff
@@ -1,43 +1,390 @@
+
+ # Mogu specific
+ warlord_male_mogu_duke:0 "Warlord"
+ warlady_female_mogu_duke:0 "Warlady"
+ clan_mogu_duchy:0 "Clan"
+
+ ### Jinyu ###
+ # Feudal / Clan
+ joshu_male_jinyu_baron:0 "Joshu"
+ joshu_female_jinyu_baron:0 "Joshu"
+ shiro_jinyu_barony:0 "Shiro"
+ kokujin_male_jinyu_count:0 "Kokujin"
+ kokujin_female_jinyu_count:0 "Kokujin"
+ kori_jinyu_county:0 "Kori"
+ daimyo_male_jinyu_duke:0 "Daimyo"
+ daimyo_female_jinyu_duke:0 "Daimyo"
+ clan_jinyu_duchy:0 "Clan"
+ shogun_male_jinyu_king:0 "Shogun"
+ shogun_female_jinyu_king:0 "Shogun"
+ shogunate_jinyu_kingdom:0 "Shogunate"
+
+ # Republic
+ toshiyori_male_jinyu_baron:0 "Toshiyori"
+ toshiyori_female_jinyu_baron:0 "Toshiyori"
+ chimachi_jinyu_baron:0 "Chimachi"
+ yuso_male_jinyu_count:0 "Yuso"
+ yuso_female_jinyu_count:0 "Yuso"
+ machi_jinyu_county:0 "Machi"
+ odoshiyori_male_jinyu_duke:0 "Odoshiyori"
+ odoshiyori_female_jinyu_duke:0 "Odoshiyori"
+ omachi_jinyu_duchy:0 "Omachi"
+ sokan_male_jinyu_king:0 "Sokan"
+ sokan_female_jinyu_king:0 "Sokan"
+ domain_jinyu_kingdom:0 "Domain"
+ shushoo_male_jinyu_emperor:0 "Shushoo"
+ shushoo_female_jinyu_emperor:0 "Shushoo"
+ kyowa_jinyu_empire:0 "Kyowa"
+
+ # Theocracy
+ juji_male_jinyu_baron:0 "Juji"
+ juji_female_jinyu_baron:0 "Juji"
+ chitera_jinyu_baron:0 "Chitera"
+ saito_male_jinyu_count:0 "Saito"
+ saito_female_jinyu_count:0 "Saito"
+ tera_jinyu_county:0 "Tera"
+ dai_sojo_male_jinyu_duke:0 "Dai-Sojo"
+ dai_sojo_female_jinyu_duke:0 "Dai-Sojo"
+ otera_jinyu_duchy:0 "Otera"
+ hossu_male_jinyu_king:0 "Hossu"
+ hossu_female_jinyu_king:0 "Hossu"
+ kyodan_jinyu_kingdom:0 "Kyodan"
+ haku_male_jinyu_emperor:0 "Haku"
+ haku_female_jinyu_emperor:0 "Haku"
+ daihoshu_jinyu_empire:0 "Daihoshu"
+
+ ### Goblin ###
+ # Feudal / Republic
+ trade_baron_male_goblin_count:0 "Trade Baron"
+ trade_baroness_female_goblin_count:0 "Trade Baroness"
+ high_barony_goblin_county:0 "High Barony"
+ mogul_male_goblin_duke:0 "Mogul"
+ mogul_female_goblin_duke:0 "Mogul"
+ company_goblin_duchy:0 "Company"
+ trade_prince_male_goblin_king:0 "Trade Prince"
+ trade_princess_female_goblin_king:0 "Trade Princess"
+ cartel_goblin_kingdom:0 "Cartel"
+ president_male_goblin_emperor:0 "President"
+ president_female_goblin_emperor:0 "President"
+ coalition_goblin_empire:0 "Coalition"
+
+ ### Centaur ###
+ # Feudal
+ khan_male_centaur_king:0 "Khan"
+ khan_female_centaur_king:0 "Khan"
+ khanat_centaur_kingdom:0 "Khanat"
+ khagan_male_centaur_emperor:0 "Khagan"
+ khagan_female_centaur_emperor:0 "Khagan"
+ khaganat_centaur_empire:0 "Khaganat"
+
+ ### Taurens ###
+ # Feudal/Clans/Tribal
+ chieftain_male_tauren_count:0 "Chieftain"
+ chieftain_female_tauren_count:0 "Chieftain"
+ high_chieftain_male_tauren_duke:0 "High Chieftain"
+ high_chieftain_female_tauren_duke:0 "High Chieftain"
+ great_chieftain_male_tauren_king:0 "Great Chieftain"
+ great_chieftain_female_tauren_king:0 "Great Chieftain"
+ patriarch_male_tauren_emperor:0 "Patriarch"
+ matriarch_female_tauren_emperor:0 "Matriarch"
```

Comment:
Why the name of county, duchy, kingdom, etc. aren't set?

Reply:
done in 4971a71b1